### PR TITLE
Tutorial binder

### PR DESCRIPTION
**Pull request by egede**
Update of the User guide.

### CODE REVIEW

File: doc/UserGuide/UsingDifferentBackends.rst [hunk 1]
```diff
@@ -14,45 +12,40 @@ to get a full list, use:
...
-``config.Local``.
+``config.Local``. You can quit Ganga and restart and the Local job will still run in the background.
 
 Batch Backends
 --------------
 
-Ganga supplies backend objects for most of the major batch systems around - LSF, PBS and  Condor. You should
-obviously use the one that is relevant to the system you are running on. Typical usage is detailed below though
-as with all these, you can get more help using ``help(<backend>)`` and ``config.<backend>``:
+Ganga supplies backend objects for most of the major batch systems around - Condor, Slurm, SGE, LSF, PBS and  Condor. You should obviously use the one that is relevant to the system you are running on. Typical usage is detailed below though as with all these, you can get more help using ``help(<backend>)`` and ``config.<backend>``. Sometimes a local installation requires that small changes are made to the configuration. Look in the relevant section of the ``~/.gangarc`` file.
```
Comment (mesmith75) [inline doc/UserGuide/UsingDifferentBackends.rst]: 2 Condors

**Review by alexanderrichards — DISMISSED** [review]

**Review by mesmith75 — CHANGES_REQUESTED** [review]

File: doc/UserGuide/UsingDifferentBackends.rst (hunk 1 above)
Comment (alexanderrichards) [reply doc/UserGuide/UsingDifferentBackends.rst]: good spot! I missed that

File: doc/UserGuide/InstallAndBasicUsage.rst [hunk 2]
```diff
@@ -216,41 +202,55 @@ You can also select specific info about the job object, e.g. the application tha
...
-.. literalinclude:: ../../ganga/GangaCore/test/GPI/TutorialTests.py
-    :start-after: # -- INSTALLANDBASICUSAGE EXECFILE START
-    :end-before: # -- INSTALLANDBASICUSAGE EXECFILE STOP
-    :dedent: 8
+.. code-block:: bash
+
+    [centos7] ~ % cat >> myfile.py
+    j = Job()
+    j.submit()
+    ^D
```
Comment (mesmith75) [inline doc/UserGuide/InstallAndBasicUsage.rst:242]: Is the `^D` intended?

Comment (egede) [reply doc/UserGuide/InstallAndBasicUsage.rst:242]: Yes, it is the EOF when creating a file.

File: doc/UserGuide/UsingDifferentBackends.rst (hunk 1 above)
Comment (egede) [reply doc/UserGuide/UsingDifferentBackends.rst]: Fixed

**Review by mesmith75 — APPROVED** [review]

**Pull request merged**